Entities/Items/Crate/MakeCrate.as

```diff
@@ -10,7 +10,7 @@ shared CBlob@ server_MakeCrate(string blobName, string inventoryName, int frameI
 		crate.server_setTeamNum(team);
 		crate.setPosition(pos);
 		crate.set_string("packed", blobName);
-		crate.set_string("packed name", inventoryName);
+		crate.set_string("packed name", getTranslatedString(inventoryName));
```

Comment:
Then you also need to update `Crate.as` which uses this property:

```angelscript
			this.setInventoryName("Crate with " + this.get_string("packed name"));
```

And shouldn't we rather use `getTranslatedString` wherever the `"packed name"` property is being read?

Reply:
Done